Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6996,27 +6998,19 @@ class BuildPlanSwiftBuildTests: BuildPlanTestCase {
         try await super.testDuplicateProductNamesWithNonDefaultLibsThrowError()
     }
 
-    override func testTargetsWithPackageAccess() async throws {
-        throw XCTSkip("Skip until swift build system can support this case.")
-    }
-
-    override func testTestModule() async throws {
-        throw XCTSkip("Skip until swift build system can support this case.")
-    }
-
     override func testPackageNameFlag() async throws {
-#if os(Windows)
+        #if os(Windows)
         throw XCTSkip("Skip until there is a resolution to the partial linking with Windows that results in a 'subsystem must be defined' error.")
```

Comment:
question: does a GitHub issue exist for this Windows issue?  if so, can we update the comment to include a link to it? or maybe add the issue as a code comment.

Reply:
I think this is the issue we are looking for #8380, as it mentions this exact same error `"subsystem must be defined' error."`. I will add a comment about it...